challenges/headstorm-bec-jehillert/.notes/README - Applicant's Notes.txt

```diff
@@ -0,0 +1,213 @@
+TO DO
+———————————————————————————————————————————————————————————————————————————————
+  Make sure error from bodyparser indicates the correct information.
+
+  X Make sure content of rest api tutorial is not something that is going to change what you thought the assignment was about.
+
+  Provide API documentation.  Specifically, instructions on the form the data should be in (e.g., array or object), json, etc.
+
+  Changes to make to variable names:
+    sorted_list_val_id ==> sorted_val_id
+    unsorted_list_val_id ==> unsorted_val_id
+
+QUESTIONS
+———————————————————————————————————————————————————————————————————————————————
+  Hi Gianna,
+
+  Would it be possible to pass the following questions on to the developers in charge of the Back End Challenge?  Brief answers would be fine (and greatly appreciated)
+
+  Thanks much,
+
+  John
+
+  -----------------------------------------------
+
+  QUESTION 1: Should we store the data as a table row containing a JSON, a table of parsed number values (each number a row), or is there another preferred structure. My [imperfect] understanding is that a table of values is strongly preferred, at least in the context of relational databases.
+
+  QUESTION 2:  May we store the data in unsorted form, and handle sorting at the retrieval stage?  My understanding is that adding new values to the bottom rows of a table as the values come in, and then handling sorting when the data is needed, is preferred.
```

Comment:
Your assumption makes sense to me, and using a relational DB with each number in a row would make getting the sorted results back easy. Go ahead and store it unsorted.